src/model/ai/evaluation/GameOverEvaluator.java

```diff
@@ -0,0 +1,33 @@
+package model.ai.evaluation;
+
+import model.Board;
+import model.PieceColor;
+
+public class GameOverEvaluator extends BoardEvaluator {
+
+    public GameOverEvaluator() {
+        super();
+    }
+
+    public GameOverEvaluator(double weight) {
+        super(weight);
+    }
+
+    @Override
+    public double evaluateBoard(Board theBoard, PieceColor color) {
+        double value = 0.0;
+
+        PieceColor winner = theBoard.determineWinner();
+
+        if (winner == null) {
```

Comment:
We'll use a similar refactor as the the refactor of removing the determineWinner() method call in GameManager and using board.isEndState(PieceColor color) calls instead. this should make the management of null PieceColor moot at this juncture.